vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -194,19 +194,21 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.27.14-hotfix.20240712/windowszip/v1.27.14-hotfix.20240712-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.27.15-hotfix.20240712/windowszip/v1.27.15-hotfix.20240712-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.27.16/windowszip/v1.27.16-1int.zip",
-        "https://acs-mirror.azureedge.net/kubernetes/v1.28.11-hotfix.20240712/windowszip/v1.28.11-hotfix.20240712-1int.zip",
+        "https://acs-mirror.azureedge.net/kubernetes/v1.27.100-akslts/windowszip/v1.27.100-akslts-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.28.12/windowszip/v1.28.12-1int.zip",
```

Comment:
we have more than 2 patch for these minors, if we hit VHD size issues, we should reduce these.

Reply:
I agree.
`For supported minor versions: Keep 4 patch versions` this principle was written 3 years ago, it may not be necessary to comply now, we have too many minor versions